docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -11,46 +11,64 @@ other resources such as Kubernetes clusters and applications coming in the futur
 ## Installation & upgrade
 
 Head over to the [Downloads](https://goteleport.com/download/) page to download the most recent
-version of Teleport Connect. As of now, Teleport Connect supports macOS only. Double-click the
-downloaded `.dmg` file and drag the Teleport Connect icon to the Applications folder.
+version. Teleport Connect supports macOS, Linux, and Windows.
+
+<Tabs>
+<TabItem label="macOS">
+Double-click the downloaded `.dmg` file and drag the Teleport Connect icon to the Applications folder.
 
 To upgrade Teleport Connect to a newer version, drag the new version to the Applications folder.
+</TabItem>
+<TabItem label="Linux">
+Download the DEB (Debian-based distros) or RPM (RHEL-based distros) package and install it using
+your package manager. Repeat the process for in-place upgrades.
+
+You can also download the project as a `tar.gz` file to extract and run it in place.
+</TabItem>
+<TabItem label="Windows">
+Download and run the installer `.exe` file. It will install and open Teleport Connect without
+further user input.
+
+Repeat the process with newer versions to upgrade.
+</TabItem>
+<TabItem label="From Source">
+</TabItem>
```

Comment:
Do we want to leave this empty or can we outright remove it? It doesn't seem to have an effect on the UI.

Reply:
This was a relic from when I was building the doc and thought that the tarball was source code and not a binary for Linux. Will remove:

```suggestion
```